### DISCUSSION TITLE
Acess to files

### DISCUSSION THREAD

**Discussion by Dart1999** [coobbi/Jre4android#8] [Q&A]
![Screenshot_20241026-185822_Jre4Android-Java Runtime Environment](https://github.com/user-attachments/assets/1ce68f21-6d80-443e-9526-9bf69da1e762)
![Screenshot_20241026-185814_Jre4Android-Java Runtime Environment](https://github.com/user-attachments/assets/165c3271-0711-4611-a778-163cece5c84c)
please tell me how to acess these through android without root or pc

**Comment by coobbi** [comment]
@Dart1999 Could you be more specific? Which specific file do you want to access?

**Reply by Dart1999** [reply]
The classpath file is there a way to get acess to that file because i cant find it here unless i use a modloader.jar is installing mod folder in classpath file 
![Screenshot_20241028-024225_Jre4Android-Java Runtime Environment](https://github.com/user-attachments/assets/4e2fb4d3-590b-4b45-b874-0ff48b1a538d)
![Screenshot_20241028-024216_Jre4Android-Java Runtime Environment](https://github.com/user-attachments/assets/cb23d093-2ba5-4d54-9ab1-f9b19eb6db74)
![Screenshot_20241028-024149_Jre4Android-Java Runtime Environment](https://github.com/user-attachments/assets/a9c3152b-39d0-44dd-92b6-0a40519822e0)
![Screenshot_20241028-024143_Jre4Android-Java Runtime Environment](https://github.com/user-attachments/assets/11a7f816-ff7d-4c73-b13a-7b251d53a98f)
![Screenshot_20241028-024133_Jre4Android-Java Runtime Environment](https://github.com/user-attachments/assets/dcbc0462-6040-4749-961f-fd70983c5466)
![Screenshot_20241028-024125_Jre4Android-Java Runtime Environment](https://github.com/user-attachments/assets/82f9e66e-b86b-4e6a-919f-7b9995d3bf4b)

**Comment by coobbi** [comment]
@Dart1999 
This app change some codes which can add all type files to classpaht(not only .jar file), try this app
[Download apk](https://github.com/coobbi/Jre4android/releases/download/pre_free_release_0.9.3/app_launcher-free-release.apk)

Due to Android permissions, the app cannot directly access the file through code, so you need to copy the file to the internal file directory of the jre4android application so that you can access it.

1. Setting->Setting->Add jars to classpath ->click right-corner button ->select your right files
![1111](https://github.com/user-attachments/assets/ac9cbb1e-9cac-4c0e-b539-212b61802f5f)

2. open your jar(modloader.jar) 
![124](https://github.com/user-attachments/assets/ed6a5849-50e8-45d0-9d11-ec1c29d29419)

3. select file from classpath(choose all type)
![123](https://github.com/user-attachments/assets/799515b6-da51-45d0-8624-e7b88b21ea11)

**Comment by coobbi** [comment]
@Dart1999 New app can be updated from Google Play

**Comment by Dart1999** [comment]
Is there a way to copy & paste from classpath in the app?
